.github/CHANGELOG.md

```diff
@@ -17,12 +17,13 @@
 * Adds support for multi-qubit expectation values in `default.qubit`.
   [#202](https://github.com/XanaduAI/pennylane/pull/202)
 
-* Organise templates into submodules [#195](https://github.com/XanaduAI/pennylane/pull/195).
+* Organize templates into submodules [#195](https://github.com/XanaduAI/pennylane/pull/195).
   This included the following improvements:
 
-  - Distinguish embedding templates, layer templates and parameter templates
+  - Distinguish embedding templates, layer templates, and parameter templates.
+    Parameter templates are contained within the new submodule `pennylane.init`.
```

Comment:
Are we calling them `parameter templates`? Probably better to just say we added new random initialization functions supporting the templates

Reply:
That sounds a lot better, I've made the suggested update.